core/got-test-client.js

```diff
@@ -3,4 +3,4 @@
 const got = require('got')
 
 // https://github.com/nock/nock/issues/1523
-module.exports = got.extend({ retry: 0 })
+module.exports = got.extend({ retry: 0, throwHttpErrors: false })
```

Comment:
This client is imported in a lot of places, and I'm not sure I'm convinced `throwHttpErrors: False` should be the default everywhere. (Whereas `retry: 0` avoids a lot of surprises, especially when using mock requests.)

Would it work to define this in the file where it's needed?

Reply:
Sure. I moved `throwHttpErrors` to the test which requires it.